src/CsvTextEditor/Commands/FileOpenInTextEditorCommandContainer.cs

```diff
@@ -13,11 +15,36 @@ namespace CsvTextEditor
 
     public class FileOpenInTextEditorCommandContainer : FileOpenInExternalToolCommandContainerBase
     {
+        private readonly IFileExtensionService _fileExtensionService;
+        private readonly IProcessService _processService;
+        private readonly IConfigurationService _configurationService;
 
         public FileOpenInTextEditorCommandContainer(ICommandManager commandManager, IProjectManager projectManager, IFileExtensionService fileExtensionService,
-            IFileService fileService, IProcessService processService)
-            : base(Commands.File.OpenInTextEditor , "txt", commandManager, projectManager, fileExtensionService, fileService, processService)
+            IFileService fileService, IProcessService processService, IConfigurationService configurationService)
+            : base(Commands.File.OpenInTextEditor, "txt", commandManager, projectManager, fileExtensionService, fileService, processService)
         {
+            Argument.IsNotNull(() => fileExtensionService);
+            Argument.IsNotNull(() => configurationService);
+            Argument.IsNotNull(() => processService);
+
+            _fileExtensionService = fileExtensionService;
+            _processService = processService;
+            _configurationService = configurationService;
+        }
+
+        protected override void Execute(object parameter)
+        {
+            var externalToolPath = _configurationService.GetLocalValue<string>(Configuration.CustomEditor);
+
+            if (externalToolPath != null)
```

Comment:
Since this code is nearly identical, maybe it can be rewritten as:

```
var toolpath = externalToolPath ?? _fileExtensionService.GetRegisteredTool("txt"); 
_processService.StartProcess(toolPath, _projectManager.ActiveProject.Location);
```

Reply:
Thanks Geert, no problem, just done a pull request. Sorry it took a couple of days, I keep expecting to get notifications from GitHub, but I don't!

Reply:
@theescapers It could be in your "non-focused" inbox. But I use this instead:

https://octobox.io/

Just keep it pinned in your favorite browser and you can check that a few times a day.